relayer-cli/Cargo.toml

```diff
@@ -26,11 +26,11 @@ telemetry   = ["ibc-relayer/telemetry", "ibc-telemetry"]
 rest-server = ["ibc-relayer-rest"]
 
 [dependencies]
-ibc              = { version = "0.12.0-rc.0", path = "../modules", features = ["std", "clock"] }
-ibc-relayer      = { version = "0.12.0-rc.0", path = "../relayer" }
+ibc              = { version = "0.12.0", path = "../modules", features = ["std", "clock"] }
+ibc-relayer      = { version = "0.12.0", path = "../relayer" }
 ibc-proto        = { version = "0.16.0-rc.0", path = "../proto" }
```

Comment:
Should `ibc-proto`'s version be bumped (Just for consistency's sake).

Reply:
Duh, fixed!